drivers/ieee802154/ieee802154_cc13xx_cc26xx.c

```diff
@@ -0,0 +1,735 @@
+/*
+ * Copyright (c) 2019 Brett Witherspoon
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <errno.h>
+#include <kernel.h>
+#include <misc/byteorder.h>
+#include <net/ieee802154_radio.h>
+#include <net/ieee802154.h>
+#include <net/net_if.h>
+#include <net/net_pkt.h>
+#include <random/rand32.h>
+#include <string.h>
+#include <sys_io.h>
+
+#define LOG_LEVEL CONFIG_IEEE802154_DRIVER_LOG_LEVEL
```

Comment:
It would be better to place the logging settings before all the other include files (see examples in other .c files). The reason being that if some include files have logging in  inline functions, the logging is registered properly.

Reply:
Ah. I was wondering why this was done. Fixed.